### PR TITLE
point to updated article

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2511

as advised by Daniel Simpson

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-antivirus/windows-defender-antivirus-in-windows-10.md [hunk 1]
```diff
@@ -42,23 +42,8 @@ You can configure and manage Windows Defender Antivirus with:
...
-- [Updates to how the block at first sight feature can be configured](configure-block-at-first-sight-windows-defender-antivirus.md)
-- [The ability to specify the level of cloud-protection](specify-cloud-protection-level-windows-defender-antivirus.md)
-- [Windows Defender Antivirus protection in the Windows Security app](windows-defender-security-center-antivirus.md)
-
-We've expanded this documentation library to cover end-to-end deployment, management, and configuration for Windows Defender Antivirus, and we've added some new guides that can help with evaluating and deploying Windows Defender AV in certain scenarios:
-- [Evaluation guide for Windows Defender Antivirus](evaluate-windows-defender-antivirus.md)
-- [Deployment guide for Windows Defender Antivirus in a virtual desktop infrastructure environment](deployment-vdi-windows-defender-antivirus.md)
-
+>[!NOTE]
+For more information regarding what's new in each Windows version, please refer to [What's new in Microsoft Defender ATP](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/whats-new-in-microsoft-defender-atp)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-antivirus/windows-defender-antivirus-in-windows-10.md]:
```suggestion
For more information regarding what's new in each Windows version, please refer to [What's new in Microsoft Defender ATP](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/whats-new-in-microsoft-defender-atp).
```

**Comment by mypil** [comment]
@VLG17 - please merge the commit for the copy/edit review done by Johan. Please also check on the branch conflict. Thanks.

**Comment by VLG17** [comment]
I cannot resolve the branch conflict.

When going on "Resolve conflicts" it leads me to a page where the option "Mark as resolved" is grayed out, and i don't have any other option there...

image

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #2511.

Please take note that there's a branch conflict that @VLG17 tried to resolve but could not so please advise how to proceed. 

Thank you.

**Comment by martyav** [comment]
@VLG17, I just had a look at it. 

From the browser, you need to edit the commit by clicking into the text field, then removing the lines containing >>>>>> etc. & <<<<<< etc., before the 'resolve' button will go from grayed out to enabled.

**Comment by VLG17** [comment]
> @VLG17, I just had a look at it.
> 
> From the browser, you need to edit the commit by clicking into the text field, then remove the lines containing >>>>>> etc. & <<<<<< etc., before the 'resolve' button will go from grayed out to enabled.

@martyav thank you, worked like a charm, although i don't understand how that got added there (i definitely did not add that).

**Comment by martyav** [comment]
@VLG17 When there's a merge conflict, that means Git isn't sure what to update inside the file. So, it'll mark off the parts it isn't sure about with angle brackets. One section will be your changes, and the other section will be what's currently in the branch you're merging into. The coder who handles the conflict picks what to keep, then removes the markup and commits the file to resolve the conflict. I hope that makes sense.

**Pull request merged**